app/binders/propertylinks/ExternalPropertyLinkManagementSortField.scala

```diff
@@ -24,4 +26,19 @@ object ExternalPropertyLinkManagementSortField extends Enumeration {
   val AGENT = Value("AGENT")
   val STATUS = Value("STATUS")
   val REPRESENTATION_STATUS = Value("REPRESENTATION_STATUS")
+
+  implicit val queryStringBindable: QueryStringBindable[ExternalPropertyLinkManagementSortField] =
+    new QueryStringBindable[ExternalPropertyLinkManagementSortField] {
+      override def bind(
+            key: String,
+            params: Map[String, Seq[String]]
+      ): Option[Either[String, ExternalPropertyLinkManagementSortField]] =
+        params.get(key).collect {
+          case Seq(s) =>
+            ExternalPropertyLinkManagementSortField.values.find(_.toString == s).toRight("invalid value")
```

Comment:
can this just be `values.find(_.toString == s).toRight("invalid value")` ?

Reply:
could the error be a bit more specific? "invalid sort field parameter" or something like that...